apps/rules/metadata-commit/src/lambda.types.ts

```diff
@@ -0,0 +1,29 @@
+import type { NonEmptyString } from '@carrot-fndn/shared/types';
+
+import { Octokit } from 'octokit';
+
+export interface Rule {
+  commit_hash: string;
+  file_checksum: string;
+  source_code_url: string;
+}
+
+export interface Metadata {
+  'commit-hash': string;
+  'file-checksum': string;
+  'rule-name': string;
+  'source-code-url': string;
+}
+
+export interface EnvironmentVariables {
+  ENVIRONMENT: NonEmptyString;
+  GH_OCTOKIT_TOKEN: NonEmptyString;
+  GH_OWNER: NonEmptyString;
+  GH_REPO: NonEmptyString;
+  REFERENCE_BRANCH: NonEmptyString;
+}
+
+export type Rules = { rules: Record<string, Record<string, string>> };
```

Comment:
como só tem uma propriedade, será que não valeria a pena passar ela pra raíz?
```suggestion
export type Rules = Record<string, Record<string, string>>;
```